Samples/VirtualCamera/README.md

```diff
@@ -71,9 +69,24 @@ This should include the mediasource dll, the executable that is responsbile for
     3. From the popup windows select the executable with the uninstall routine.  
     In this example it is VirtualCamera_ConsoleApp.exe /uninstall
 
-## VirtualCamera_UWP
+## VirtualCameraManager_App
 ----
--- TBD -- 
+### To build:
+1. Build the *VirtualCameraManager_WinRT* project
+2. Build the *VirtualCameraManager_App*
+3. Either launch the app right away (right click on the *VirtualCameraManager_App* project and select *deploy*) or create an app package you can share around and sideload (right click on the *VirtualCameraManager_App* project and select *publish*)
+
+### What the app does:
+- When you launch the app, it will contain a barebone UI with a ➕ sign button at the top. This opens a dialog to create a new virtual camera.
+- If you create a virtual camera with the *Session* lifetime, this virtual camera would then be removed upon closing the app. However, if you create a virtual camera with the *System* lifetime, it will remain functional until the system reboots or you manually remove it by either relaunching the app or manually removing it from the Windows Settings (Devices &rarr; Camera) page. 
+The app caches the set of virtual camera it creates in a configuration file upon closure and invertedly reloads the virtual camera that were previously created (with a lifetime = *System*) in a previous session.
+(see the methods ```LoadConfigFileAsync()``` and ```WriteToConfigFileAsync()``` in *App.xaml.cs* in the *VirtualCameraManager_App* project).
+If a user would like to recover virtual cameras created in the past after rebooting their system, then relaunching the app would cause that. You could automate this process if you wish on behalf of the user using a service for example that would start the app upon startup of the system.
+- Once a virtual camera is created, if hooked into an existing camera and that camera supports the eye gaze correction control, then the UI element for it will be available. This is to show how a virtual camera can relay existing control interactions if they are supported onto the original camera it is hooked into. Similarly, if you did not hook into an existing camera, there will be a UI element to interact with the color of the synthesized frame via a custom control defined in the virtual camera (see *VirtualCameraMediaSource.h* in the *VirtualCameraMediaSource* project).

```

Comment:
>Once a virtual camera is created, if hooked into an existing camera and that camera supports the eye gaze correction control [](http://example.com/codeflow?start=2&length=124)

Could rephrase: If a virtual camera is created from an existing camera that supports the eye gaze correction control, ...